src/main/resources/db/migration/V9__varighet.sql

```diff
@@ -0,0 +1,5 @@
+alter table sykefravar_statistikk_virksomhet add column varighet varchar(1);
+
+DROP INDEX orgnr_arstall_kvartal__index;
```

Comment:
Er det nødvendig å slette orgnr/årstall/kvartal indeksen? Hvis vi vil fortsette å kjøre queries som henter data basert på disse tre felter da blir indeksen fortsatt relevant.
Og vi kan ha flere indekser på en tabell (ulempen med "for mange" indekser er tregere insert/update, men det spiller ikke så stor rolle for oss, vi har bare insert 4 ganger i året)

Reply:
For ordens skyld: Trenger ikke slette indeksen, men vi dropper "unique"-constraint.